data/json/npcs/refugee_center/surface_visitors/NPC_old_guard_representative.json

```diff
@@ -244,7 +244,7 @@
     "difficulty": 2,
     "value": 10000,
     "start": {
-      "assign_mission_target": { "om_terrain": "exodii_base_x0y3z0", "om_special": "exodii_base", "reveal_radius": 5, "search_range": 400 },
+      "assign_mission_target": { "om_terrain": "exodii_base_x0y3z0", "om_special": "exodii_base", "reveal_radius": 5, "search_range": 1000 },
```

Comment:
```suggestion
      "assign_mission_target": { "om_terrain": "exodii_base_x0y3z0", "om_special": "exodii_base", "reveal_radius": 5, "search_range": 800 },
```

Reply:
Over 800 seems to result in more complaints of potatoes freezing when missions fire.